src/shared/components/FormComponents/InputController/Input/Input.tsx

```diff
@@ -97,7 +97,7 @@ export const Input = <T extends FieldValues>({
   useEffect(() => {
     if (!withDebounce || !inputRef.current || inputRef.current?.value === String(value)) return;
     inputRef.current.value = value;
-  });
+  }, [withDebounce, inputRef.current, value]);
```

Comment:
what if value will be the same twice or more times (in case value in input set programmatically with custom onChange)?